apis/placement/v1beta1/resourcesnapshot_types.go

```diff
@@ -39,8 +42,8 @@ const (
 // Its spec is immutable.
 // We may need to produce more than one resourceSnapshot for all the resources a ResourcePlacement selected to get around the 1MB size limit of k8s objects.
 // We assign an ever-increasing index for each such group of resourceSnapshots.
-// The name convention of a clusterResourceSnapshot is {CRPName}-{resourceIndex}(-{subindex})*
-// where the name of the first snapshot of a group has no subindex part so its name is {CRPName}-{resourceIndex}.
+// The name convention of a clusterResourceSnapshot is {CRPName}-{resourceIndex}-{snapshot|-{subindex}}
```

Comment:
Nit, naming convention?

Reply:
Also a nit: since the special case has been explained, here we can just use {CRPName}-{resourceIndex}-{subindex} to avoid confusion.